app/src/main/java/org/oppia/android/app/topicdownloaded/TopicDownloadedFragment.kt

```diff
@@ -0,0 +1,36 @@
+package org.oppia.android.app.topicdownloaded
+
+import android.content.Context
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import org.oppia.android.app.fragment.InjectableFragment
+import javax.inject.Inject
+
+/** Fragment that contains topic downloaded view. */
+class TopicDownloadedFragment : InjectableFragment() {
+
+  @Inject
+  lateinit var topicDownloadedFragmentPresenter: TopicDownloadedFragmentPresenter
+
+  override fun onAttach(context: Context) {
+    super.onAttach(context)
+    fragmentComponent.inject(this)
+  }
+
+  override fun onCreateView(
+    inflater: LayoutInflater,
+    container: ViewGroup?,
+    savedInstanceState: Bundle?
+  ): View? {
+    val internalProfileId = arguments?.getInt(PROFILE_ID_ARGUMENT_KEY, -1)!!
+    val topicId = arguments?.getString(TOPIC_ID_ARGUMENT_KEY)!!
```

Comment:
Prefer adding checkNotNull(){} calls here with more contextual errors in case a null is actually encountered (see other checkNotNull cases in onCreateView() calls for reference).